src/app/errorReports/newErrorReportForm/NewErrorReportForm.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
```

Comment:
"NewErrorReportForm" sounds like a reasonable name for the component, but for the directory, doesn't "errorReport" convey basically the same meaning as "newErrorReportForm"?
And arguably, the Thanks component is part of the form as well (especially in the designs, where it's just a modal on top of it), so the additional subdirectory might not really be needed.

Reply:
Yeah, I was replicating the comments directory structure with this.
I can flatten the `errorReports` directory, which should make things simpler when I integrate into the `find` flow.